build/static_dist.js

```diff
@@ -113,6 +113,8 @@ function addPackages(siteConfig, buildFolder) {
 		return readFile(path.join(fullBuildFolderPath, "package.json")).then(function(packageContents){
 			var json = JSON.parse(packageContents);
 
+			json = _.merge(json || {}, siteConfig.html.package);
```

Comment:
not sure where ... but it would be good to document that .dependencies overwrites `.package.`